MOxUnit/initTestSuite.m

```diff
@@ -65,3 +65,10 @@
             end
         end
     end
+
+    % If the user didn't request an output, immediately execute the test and
+    % display its result
+    if ~nargout
+        disp(run(test_suite));
+        clear test_suite;
```

Comment:
Since this script is, in all typical use cases, used within a function: can we remove the "clear" statement? The garbage collector would take care of cleaning up the test_suite. Or is there a use case I am missing?